pwndbg/lib/common.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import annotations
+
+
+# common functions
+def addressify_common(arg: str) -> int:
```

Comment:
```suggestion
def addressify_common(arg: str) -> int:
```

Maybe: unhex? unhexlify? or just hex2ptr, heh (its like binascii.unhexlify, but better, lol)